tox.ini

```
@@ -23,16 +23,16 @@ passenv =
   RIOTBASE
   SSH_AUTH_SOCK
   SSH_AGENT_PID
-  APPKEY
   NWKSKEY
   APPSKEY
-  LORAWAN_DL_KEY
+  DEVADDR
+  DEVEUI
+  APPEUI
+  APPKEY
+  TTN_DL_KEY
```

The renamed ones also need to be changed in RIOT's `release-tests` workflow, once merged. Maybe it is based that for the migration period we allow both names

```suggestion
  DEVADDR
  DEVEUI
  APPEUI
  APPKEY
  LORAWAN_DL_KEY
  TTN_DL_KEY
```


... (cont.)